hasher-matcher-actioner/hmalib/models.py

```diff
@@ -60,6 +60,9 @@ def to_dynamodb_dict(self) -> dict:
     def from_dynamodb_dict(cls, d: dict) -> "Label":
         return cls(d["K"], d["V"])
 
+    def __eq__(self, another_label: object):
```

Comment:
nit: another_label will be of type 'Label', not object.

Because the type is not materialized yet, mypy will allow you to do:
```python
def __eq__(self, other: "Label"):
```

Note how we are using quotes around Label.

Reply:
@schatten That was definitely a miss (you'd helped me with types and I didn't carry that learning to this module). Are you saying we _can_ use the quoted type name, or that we _should_?